scipy/signal/tests/test_ltisys.py

```diff
@@ -410,7 +412,13 @@ def test_multioutput(self):
         assert_allclose(b_all, np.vstack((b0, b1, b2)), rtol=1e-13, atol=1e-14)
 
 
-class TestLsim:
+class _TestLsimFunctions:
+    decimal = 7
```

Comment:
It seems you've taking this from the signature of `assert_almost_equal`, so I understand it this comes from. Still, I find
```
assert_almost_equal(x, expected_x, decimal=self.decimal)
```
quite jarring - for me, decimal refers primarily to a type (python, java, various SQL flavours).

So could we rename this? I don't have particularly good names in mind, but maybe one of
* `accuracy_digits`
* `digits_accuracy`
* `digits_of_accuracy`